### PR TITLE
Fix javascript-tutorial#2098 - replace let with var in IIFE example

### PR DESCRIPTION
**Pull request by MuhammedZakir**

### CODE REVIEW

File: 1-js/06-advanced-functions/04-var/article.md [hunk 1]
```diff
@@ -277,7 +279,7 @@ In all the above cases we declare a Function Expression and run it immediately.
 
 There are two main differences of `var` compared to `let/const`:
 
-1. `var` variables have no block scope, they are visible minimum at the function level.
+1. `var` variables have no block scope, they are visibility is scoped to current function, or global, if declared outside function.
```
Comment (iliakan) [inline 1-js/06-advanced-functions/04-var/article.md:282]: Please re-read your sentence: "they are visibility is scoped"?

**Review by iliakan — CHANGES_REQUESTED** [review]
Please fix, according to the comment.

File: 1-js/06-advanced-functions/04-var/article.md (hunk 1 above)
Comment (MuhammedZakir) [reply 1-js/06-advanced-functions/04-var/article.md]: Fixed. It is "their visibility is scoped".

**Comment by iliakan** [comment]
Thanks!

**Pull request merged**